specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -500,6 +500,14 @@ export const userPreferenceDefinitions = {
       general: {
         title: preferencesText.general(),
         items: {
+          addSearchBar: defineItem<boolean>({
+            title: preferencesText.addSearchBarHomePage(),
+            description: preferencesText.addSearchBarHomePage(),
```

Comment:
if description is the same as title, then why add description at all?
either specify a different description, or remove it